src/globus_sdk/globus_app/app.py

```diff
@@ -178,6 +177,26 @@ def _initialize_login_client(
         Initializes and returns an AuthLoginClient to be used in authorization requests.
         """
 
+    def _initialize_validating_token_storage(
+        self,
+        token_storage: TokenStorage,
+        consent_client: AuthClient,
+        scope_requirements: t.Mapping[str, t.Sequence[Scope]],
+    ) -> ValidatingTokenStorage:
+        """
+        Initializes the validating token storage for the app.
```

Comment:
```suggestion
        Initialize the validating token storage for the app.
```

Reply:
"Initializes" matches the other docstrings for the other `_initialize_*` methods ([here](https://github.com/globus/globus-sdk-python/blob/main/src/globus_sdk/globus_app/app.py#L178) and [here](https://github.com/globus/globus-sdk-python/blob/main/src/globus_sdk/globus_app/app.py#L226)). I'd like to stay consistent with those.

Reply:
Ah, I see. In general, other functions' docstrings appear to use active verbs, and I recommend using active verbs in all places. I advocate to break with the other initializer functions and introduce this new function with active verb phrasing.

It's not critical to me, but I recommend always using active verbs in all places.

Reply:
I don't agree in this case (not necessarily about the overall guidance; rather about making this class's docstrings less internally consistent).

Since this is a pretty small decision (a stylistic debate over a single word in an internal docstring), you okay if I proceed as is & resolve this thread?